src/to.js

```diff
@@ -21,7 +21,9 @@ function _to(options, file) {
       common.error('no such file or directory: ' + path.dirname(file));
 
   try {
-    fs.writeFileSync(file, this.toString(), 'utf8');
+    var str = this.toString();
+    fs.writeFileSync(file, str, 'utf8');
+    return str;
```

Comment:
Would `return this;` also work? I'm refactoring the ShellString type with the intention that only strings created with `common.ShellString` will have the `.to()` and `.toEnd()` types. This is so that we can avoid modifying the String prototype.